index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <header class="header">
+        <div class="header-div">
+            <div>
+            <ul class="icons-ul">                
+                <li><img class="head-icon" src="./images/faceb.png" alt="facebookicon"></li>
+                <li><img class="head-icon" src="./images/twtr.png" alt="twitter-icon"></li>
+                <li class="link">Other language</li>
+                <li class="link">my page</li>
+                <li class="link">Logout</li>
+            </ul>
+            </div>
+            
+            <div class="bars">
+        </div>
+                 <nav>
+                    <div class="span-"><img  src="./images/download.jpg" alt="#" class="inner-backg"><p class="logo">LIVING FAITH CHURCH</p> </div>
+
+                    <ul class="nav-ul">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>
+    </header>
+    
+    <section>
+        <article class="intro">
+            <div class="menu-nav">
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+          </div>
+
+           
+<div class="mobileMenu"> 
+  
+<div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+
+<span class="closeMenu">&times;</span>
+  <ul class="container">
+        <li><a href="about.html">About</a></li>
+        <li><a href="index.html">Home</a></li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>News</li>
+ </ul>
+
+</div>
+        </article>
+
+    </section>
```

Comment:
- [ ] Please add a proper `font-size` to the options shown in your mobile menu, they currently look huge compared to the screen size:

![image](https://user-images.githubusercontent.com/52635378/164945510-9b2bf800-837d-4bf9-b3b4-02fc761c9149.png)

Reply:
done